cmd/admin-service-restart.go

```diff
@@ -89,7 +90,8 @@ func mainAdminServiceRestart(ctx *cli.Context) error {
 	fatalIf(err, "Cannot get a configured admin connection.")
 
 	// Restart the specified Minio server
-	fatalIf(probe.NewError(client.ServiceRestart()), "Cannot restart server.")
+	pErr := client.ServiceSendAction(madmin.ServiceActionValueRestart)
```

Comment:
No reason to keep it as a separate err variable.. 

```
fatalIf(probe.NewError(client.ServiceSendAction(madmin.ServiceActionValueRestart)), "Unable to restart server.")
```

Reply:
Done.